TypeScript.sublime-settings

```diff
@@ -1,5 +1,5 @@
 {
-    "auto_complete_triggers" : [ {"selector": "source.ts", "characters": "."} ],
+    "auto_complete_triggers" : [ {"selector": "source.ts", "characters": "."}, {"selector": "source.jsx", "characters": "."} ],
```

Comment:
Better change this to `source.js` and add this selector to the `TypeScriptReact.sublime-settings` file.